### PR TITLE
#34699 - Add variables and default flags

### PR DESCRIPTION
**Pull request by kjugi**
+ Add missing `!default` in our styles
+ Add variables to all styles in all components (elements,modules,views,globals)

#34699

Referenced to [this issue](https://github.com/SnowdogApps/magento2-alpaca-components/issues/94)

### CODE REVIEW

File: components/02-elements/dropdown-list/_dropdown-list.scss [hunk 1]
```diff
@@ -1,14 +1,32 @@
...
+$dropdown-list-icon-transition      : transform 0.3s, opacity 0.3s !default;
+$dropdown-list-icon-color           : $gray !default;
+$dropdown-list-icon-width           : 10px !default;
+$dropdown-list-icon-height          : 6px !default;
+$dropdown-list-icon-top             : 20px !default;
+$dropdown-list-icon-right           : 15px !default;
+$dropdown-list-icon-inner-base      : 8px !default;
+$dropdown-list-link-color           : $font-color-base !default;
+$dropdown-list-link-after-background: $white !default;
+$dropdown-list-inner-list-background: $white !default;
```
Comment (anqaka) [inline components/02-elements/dropdown-list/_dropdown-list.scss]: it's really necessary? maybe just background? Please do not multiply the variable

File: components/02-elements/dropdown-menu/_dropdown-menu.scss [hunk 2]
```diff
@@ -1,21 +1,38 @@
-$dropdown-menu-box-shadow: -5px 5px 7px 0 rgba(0, 0, 0, 0.2);
+$dropdown-menu-padding : 15px !default;
+$dropdown-menu-background : $gray-dark !default;
+$dropdown-menu-color: $white !default;
+$dropdown-menu-breakpoint-large: $screen-l !default;
```
Comment (anqaka) [inline components/02-elements/dropdown-menu/_dropdown-menu.scss]: breakpoint variable not necessary

File: components/02-elements/form/select/_select.scss [hunk 3]
```diff
@@ -1,17 +1,23 @@
-$select-border-color: #d9d9d9 !default;
-$select-padding: 10px 15px !default;
-$select-border-radius: $border-radius !default;
+$select-margin              : 10px !default;
+$select-padding             : 10px 15px !default;
+$select-border-color        : #d9d9d9 !default;
+$select-border-radius       : $border-radius !default;
+$select-list-single-border  : 1px solid $select-border-color !default;
+$select-list-dropdown-border: 1px solid $select-border-color !default;
```
Comment (anqaka) [inline components/02-elements/form/select/_select.scss]: if border it's the same, let's do one variable for border

File: components/02-elements/minicart-trigger/_minicart-trigger.scss [hunk 4]
```diff
@@ -1,17 +1,21 @@
+$minicart-trigger-padding           : 10px !default;
+
 $minicart-trigger-background        : $white !default;
 $minicart-trigger-icon-width        : 30px !default;
 
 $minicart-trigger-counter-font-size : $font-size-extra-small !default;
 $minicart-trigger-counter-background: $purple !default;
 $minicart-trigger-counter-color     : $white !default;
 
+$minicart-trigger-breakpoint-large  : $screen-l !default;
```
Comment (anqaka) [inline components/02-elements/minicart-trigger/_minicart-trigger.scss]: I wouldn't use separate breakpoints in components

File: components/02-elements/side-menu-trigger/_side-menu-trigger.scss [hunk 5]
```diff
@@ -1,10 +1,20 @@
+$side-menu-trigger-width             : 30px !default;
+$side-menu-trigger-height            : 30px !default;
+$side-menu-trigger-font-size         : $font-size-extra-small !default;
+$side-menu-trigger-text-color        : $gray !default;
+$side-menu-trigger-line-background   : $gray-dark !default;
+$side-menu-trigger-line-height       : 3px !default;
+$side-menu-trigger-line-border-radius: 3px !default;
+$side-menu-trigger-transition        : $transition-base !default;
+$side-menu-trigger-breakpoint-large  : $screen-l !default;
```
Comment (anqaka) [inline components/02-elements/side-menu-trigger/_side-menu-trigger.scss]: 🚜

File: components/02-elements/side-menu/_side-menu.scss [hunk 6]
```diff
@@ -1,35 +1,45 @@
...
+$side-menu-top             : calc(100% + 1px) !default;
+$side-menu-left            : 100% !default;
+$side-menu-width           : 300px !default;
+$side-menu-background      : $white !default;
+$side-menu-box-shadow      : 0 3px 7px 0 rgba(0, 0, 0, 0.35) !default;
+$side-menu-transition      : $transition-base !default;
+$side-menu-open-transform  : translateX(-100%) !default;
+$side-menu-link-color      : $gray-dark !default;
+$side-menu-link-hover-color: $gray !default;
+$side-menu-breakpoint-large: $screen-l !default;
```
Comment (anqaka) [inline components/02-elements/side-menu/_side-menu.scss]: IMO not necessary

File: components/02-elements/wrapper-collapsible/_wrapper-collapsible.scss [hunk 7]
```diff
@@ -1,19 +1,27 @@
+$wrapper-collapsible-padding             : 10px !default;
+$wrapper-collapsible-background          : $white !default;
+$wrapper-collapsible-icon-width          : 30px !default;
+$wrapper-collapsible-icon-height         : 30px !default;
+$wrapper-collapsible-icon-fill           : $black !default;
+$wrapper-collapsible-horizontal-transform: scaleX(1) translateX(0) !default;
+$wrapper-collapsible-breakpoint-large    : $screen-l !default;
```
Comment (anqaka) [inline components/02-elements/wrapper-collapsible/_wrapper-collapsible.scss]: IMO not necessary

File: components/02-elements/table/_table.scss [hunk 8]
```diff
@@ -1,39 +1,49 @@
...
+
+//tfoot
+$table-tfoot-background           : $white !default;
+$table-tfoot-breakpoint-background: $gray-lightest !default;
+
+//even-odd
+$table-even-background            : $gray-lightest !default;
+$table-odd-background             : $white !default;
+
+$table-breakpoint-medium          : $screen-m !default;
```
Comment (anqaka) [inline components/02-elements/table/_table.scss]: IMO breakpoints not necessary

File: components/02-elements/tab/_tab.scss [hunk 9]
```diff
@@ -1,43 +1,59 @@
+$tab-padding                    : 15px !default;
+$tab-margin                     : 15px !default;
+$tab-title-border-color         : $gray-light !default;
+$tab-title-border               : 1px solid $tab-title-border-color !default;
+$tab-title-background           : $gray-lightest !default;
+$tab-breakpoint-large           : $screen-l !default;
```
Comment (anqaka) [inline components/02-elements/tab/_tab.scss]: not necessaru

File: components/02-elements/search-trigger/_search-trigger.scss [hunk 10]
```diff
@@ -1,23 +1,31 @@
+$search-trigger-width           : 60px !default;
+$search-trigger-height          : 60px !default;
+$search-trigger-padding         : 20px 13px 10px !default;
+$search-trigger-backgrond       : $white !default;
+$search-trigger-breakpoint-large: $screen-l !default;
```
Comment (anqaka) [inline components/02-elements/search-trigger/_search-trigger.scss]: not necessary

File: components/02-elements/logo/_logo.scss [hunk 11]
```diff
@@ -1,21 +1,25 @@
+$logo-padding         : 10px 20px !default;
+$logo-breakpoint-small: $screen-s !default;
+$logo-breakpoint-large: $screen-l !default;
```
Comment (anqaka) [inline components/02-elements/logo/_logo.scss]: please, remove all breakpoints variable from compnents...

File: components/03-modules/product/_product.scss [hunk 12]
```diff
@@ -1,7 +1,20 @@
...
+$product-border                   : 1px solid $product-border-color !default;
+$product-name-color               : $font-color-base !default;
+$product-name-font-size           : $font-size-base !default;
+$product-name-brekapoint-font-size: 18px !default;
+$product-type-font-size           : $font-size-small !default;
+$product-type-color               : $gray-light !default;
+$product-details-list-font-size   : $font-size-small !default;
+$product-img-name-font-size       : 18px !default;
+$product-brekapoint-small         : $screen-s !default;
+$procuct-breakpoint-medium        : $screen-m !default;
```
Comment (anqaka) [inline components/03-modules/product/_product.scss]: not necessary

File: components/03-modules/products-list-item/_products-list-item.scss [hunk 13]
```diff
@@ -1,10 +1,25 @@
...
+$product-list-item-font-size            : $font-size-extra-small !default;
+$product-list-item-details-color        : $gray !default;
+$product-list-item-name-color           : $gray-dark !default;
+$product-list-item-price-font-size      : $font-size-small !default;
+$product-list-item-price-color          : $color-primary !default;
+$product-list-item-subtotal-font-size   : $font-size-small !default;
+$product-list-item-actions-button-width : 35px !default;
+$product-list-item-actions-button-height: 35px !default;
+$product-list-item-brekapoint-small     : $screen-s !default;
+$product-list-item-brekapoint-medium    : $screen-m !default;
```
Comment (anqaka) [inline components/03-modules/products-list-item/_products-list-item.scss]: not necessary

File: components/03-modules/search/_search.scss [hunk 14]
```diff
@@ -1,38 +1,44 @@
+$search-padding          : 15px !default;
+$search-background       : $white !default;
+$search-transition       : max-height 0.5s !default;
+$search-button-margin    : 10px !default;
+$search-breakpoint-medium: $screen-m !default;
```
Comment (anqaka) [inline components/03-modules/search/_search.scss]: not necessary

File: components/04-views/cart/_discount.scss [hunk 15]
```diff
@@ -1,32 +1,41 @@
+$discount-padding             : 15px !default;
+$discount-margin              : 15px !default;
+$discount-icon-width          : 14px !default;
+$discount-icon-height         : 14px !default;
+$discount-title-icon-transform: rotate(180deg) !default;
+$discount-transition          : $transition-base !default;
+$discount-breakpoint-medium   : $screen-m !default;
```
Comment (anqaka) [inline components/04-views/cart/_discount.scss]: not necessary

File: components/04-views/cart/_main.scss [hunk 16]
```diff
@@ -1,31 +1,34 @@
+$cart-margin           : 15px !default;
+$cart-padding          : 15px !default;
+$cart-border-color     : $gray-lighter !default;
+$cart-border           : 1px solid $cart-border-color !default;
+$cart-breakpoint-small : $screen-s !default;
```
Comment (anqaka) [inline components/04-views/cart/_main.scss]: not necessary

File: components/04-views/catalog/_catalog.scss [hunk 17]
```diff
@@ -1,37 +1,55 @@
+//product-grid
+$product-grid-padding             : 10px !default;
+$product-grid-margin              : 15px !default;
+$product-grid-breakpoint-medium   : $screen-m !default;
+$product-grid-breakpoint-large    : $screen-l !default;
```
Comment (anqaka) [inline components/04-views/catalog/_catalog.scss]: not necessary

File: components/02-elements/pager/_pager.scss [hunk 18]
```diff
@@ -1,20 +1,28 @@
...
 
     &__list {
         display: flex;
         align-items: center;
         justify-content: center;
         list-style-type: none;
-        padding: 0 15px;
-        margin: 10px 0;
+        padding: 0 $pager-padding;
+        margin: ($pager-margin - 5) 0;
```
Comment (Igloczek) [inline components/02-elements/pager/_pager.scss]: It doesn't seem to be related.

File: components/02-elements/tab/_tab.scss [hunk 19]
```diff
@@ -46,41 +62,41 @@
...
             display: none;
         }
     }
 
     &__content {
+        position: relative;
         display: none;
-        padding: 10px;
+        width: 100%;
+        padding: ($tab-padding - 5);
```
Comment (Igloczek) [inline components/02-elements/tab/_tab.scss]: It doesn't seem to be related.

File: components/02-elements/table/_table.scss [hunk 20]
```diff
@@ -1,39 +1,49 @@
-$table-padding: 15px;
-$table-radius: $border-radius;
+$table-padding                    : 15px !default;
+$table-color                      : $black !default;
+$table-font-size                  : $font-size-small !default;
+$table-radius                     : $border-radius !default;
```
Comment (Igloczek) [inline components/02-elements/table/_table.scss]: `border-radius`

File: components/02-elements/table/_table.scss [hunk 21]
```diff
@@ -1,39 +1,49 @@
...
+
+//borders
+$table-border-color               : $gray-light !default;
```
Comment (Igloczek) [inline components/02-elements/table/_table.scss]: You don't need to split it.

File: components/02-elements/table/_table.scss [hunk 22]
```diff
@@ -109,18 +119,34 @@ $table-radius: $border-radius;
...
-                border-right: 1px solid $gray-lighter;
+                background: $table-tfoot-breakpoint-background;
+                border-right: $table-border-inside;
                 text-align: left;
-                display: table-cell;
+            }
+        }
+    }
+
+    &--odd-even {
```
Comment (Igloczek) [inline components/02-elements/table/_table.scss]: Follow the rules - https://docs.snowdog.pro/web/frontend/how-to-write-better-code.html

File: components/03-modules/form/_form.scss [hunk 23]
```diff
@@ -10,57 +31,37 @@ $form-legend-line-height: 1.2 !default;
...
-            margin: 0 0 0 (-$button-icon-border-width);
+            margin: $form-button-margin;
             &--native-link {
                 margin: 0;
             }
         }
-
-    }
-
-    &__fieldset,
```
Comment (Igloczek) [inline components/03-modules/form/_form.scss]: Follow the rules - https://docs.snowdog.pro/web/frontend/how-to-write-better-code.html

File: components/03-modules/popup/_popup.scss [hunk 24]
```diff
@@ -1,15 +1,26 @@
-$popup-z-index   : 50;
-$popup-arrow-size: 21px;
-$popup-arrow-top : -10px;
+$popup-padding               : 10px !default;
+$popup-width                 : 400px !default;
+$popup-background            : $white !default;
+$popup-z-index               : 50 !default;
+$popup-box-shadow-color      : $black !default;
```
Comment (Igloczek) [inline components/03-modules/popup/_popup.scss]: It's not necessary.

File: components/03-modules/popup/_popup.scss [hunk 25]
```diff
@@ -1,15 +1,26 @@
...
+$popup-box-shadow            : 0 0 8px 0 rgba($popup-box-shadow-color, 0.2) !default;
+
+//arrow
+$popup-arrow-size            : 21px !default;
+$popup-arrow-top             : -10px !default;
+$popup-arrow-background      : $white !default;
+$popup-arrow-transform       : rotate(45deg) !default;
+$popup-arrow-box-shadow-color: $black !default;
```
Comment (Igloczek) [inline components/03-modules/popup/_popup.scss]: Not necessary

File: components/03-modules/product/_product.scss [hunk 26]
```diff
@@ -1,7 +1,20 @@
+$product-padding                  : 15px !default;
+$product-border-color             : $gray-lightest !default;
+$product-border                   : 1px solid $product-border-color !default;
+$product-name-color               : $font-color-base !default;
+$product-name-font-size           : $font-size-base !default;
+$product-name-brekapoint-font-size: 18px !default;
```
Comment (Igloczek) [inline components/03-modules/product/_product.scss]: Do breakpoints have fonts 🤔

File: components/03-modules/products-list-item/_products-list-item.scss [hunk 27]
```diff
@@ -1,10 +1,25 @@
+$product-list-item-margin               : 15px !default;
+$prodct-list-item-padding               : 15px !default;
```
Comment (Igloczek) [inline components/03-modules/products-list-item/_products-list-item.scss]: typo

File: components/03-modules/products-list-item/_products-list-item.scss [hunk 28]
```diff
@@ -1,10 +1,25 @@
...
+$product-list-item-border-color         : $gray-lighter !default;
```
Comment (Igloczek) [inline components/03-modules/products-list-item/_products-list-item.scss]: Not necessary

File: components/04-views/cart/_main.scss [hunk 29]
```diff
@@ -1,31 +1,34 @@
+$cart-margin           : 15px !default;
+$cart-padding          : 15px !default;
+$cart-border-color     : $gray-lighter !default;
```
Comment (Igloczek) [inline components/04-views/cart/_main.scss]: not necessary

File: components/04-views/cart/_totals.scss [hunk 30]
```diff
@@ -1,19 +1,27 @@
+$cart-totals-padding          : 15px !default;
+$cart-totals-margin           : 15px !default;
+$cart-totals-border-color     : #d9d9d9 !default;
```
Comment (Igloczek) [inline components/04-views/cart/_totals.scss]: Not necessary

File: components/04-views/catalog/_catalog.scss [hunk 31]
```diff
@@ -1,37 +1,55 @@
...
+
+//product-list
+$product-list-padding             : 10px !default;
+$product-list-margin              : 10px !default;
+$product-list-border-color        : $gray-light !default;
```
Comment (Igloczek) [inline components/04-views/catalog/_catalog.scss]: I don't want to reapeat myself N more time, so just remove all separate variables for border-colors

File: components/04-views/catalog/_catalog.scss [hunk 32]
```diff
@@ -1,37 +1,55 @@
...
     width: 100%;
     margin: 0;
     padding: 0;
-    justify-content: flex-start;
     list-style-type: none;
 
-    @include mq($screen-m) {
-        padding: 10px 0 10px 20px;
+    @include mq($product-grid-breakpoint-medium) {
+        padding: $product-grid-padding 0 $product-grid-padding ($product-grid-padding * 2);
```
Comment (Igloczek) [inline components/04-views/catalog/_catalog.scss]: Are you sure that in every case we would like to have a left padding that is two times wider than base padding and no left padding at all.

File: components/04-views/catalog/_catalog.scss [hunk 33]
```diff
@@ -1,37 +1,55 @@
...
-            margin-right: 15px;
+            margin-right: $product-grid-margin;
         }
 
-        @include mq($screen-m) {
+        @include mq($product-grid-breakpoint-medium) {
             flex-basis: calc(33.33% - 20px);
-            margin-bottom: 20px;
-            margin-right: 20px;
+            margin-bottom: ($product-grid-margin + 5);
```
Comment (Igloczek) [inline components/04-views/catalog/_catalog.scss]: General comment about all variables that are used in many places but with some math around it - I'm not sure about it, b/c it might be kind frustrating that the "core" value affect child elements in some, kinda random, way.

File: components/02-elements/banner/_banner.scss [hunk 34]
```diff
@@ -1,8 +1,31 @@
...
+$banner-content-background                 : $white !default;
+$banner-content-background-breakpoint-small: rgba($white, 0.6) !default;
+$banner-content-width-breakpoint-small     : 60% !default;
+$banner-content-height-breakpoint-small    : 80% !default;
+
+//content place
+$banner-top-value                          : $banner-default-position-value !default;
+$banner-bottom-value                       : $banner-default-position-value !default;
+$banner-left-value                         : $banner-default-position-value !default;
+$banner-right-value                        : $banner-default-position-value !default;
```
[the remark this answers is not in the document]
Comment (kjugi) [reply components/02-elements/banner/_banner.scss]: ok, deleted

File: components/02-elements/banner/_banner.scss [hunk 35]
```diff
@@ -1,8 +1,31 @@
...
+
+//without-image
+$banner-without-image-background           : $white !default;
+$banner-without-image-height               : 50vh !default;
+$banner-without-image-width                : 100% !default;
+
+//breakpoints
+$banner-breakpoint-large                   : $screen-xl !default;
+$banner-breakpoint-medium                  : $screen-l !default;
+$banner-breakpoint-small                   : $screen-m !default;
```
[the remark this answers is not in the document]
Comment (kjugi) [reply components/02-elements/banner/_banner.scss]: ok, will repair this

File: components/02-elements/cookie-message/_cookie-message.scss [hunk 36]
```diff
@@ -60,67 +134,4 @@
...
-            width: 10px;
-            height: 10px;
-            margin: auto;
-            top: 0;
-            left: 0;
-            right: 0;
-            bottom: 0;
-            position: absolute;
-        }
-    }
```
[the remark this answers is not in the document]
Comment (kjugi) [reply components/02-elements/cookie-message/_cookie-message.scss]: so what now?

File: components/02-elements/dropdown-menu/_dropdown-menu.scss (hunk 2 above)
Comment (kjugi) [reply components/02-elements/dropdown-menu/_dropdown-menu.scss]: deleted

File: components/02-elements/minicart-trigger/_minicart-trigger.scss (hunk 4 above)
Comment (kjugi) [reply components/02-elements/minicart-trigger/_minicart-trigger.scss]: deleted

File: components/02-elements/search-trigger/_search-trigger.scss (hunk 10 above)
Comment (kjugi) [reply components/02-elements/search-trigger/_search-trigger.scss]: deleted

File: components/02-elements/side-menu-trigger/_side-menu-trigger.scss (hunk 5 above)
Comment (kjugi) [reply components/02-elements/side-menu-trigger/_side-menu-trigger.scss]: deleted

File: components/02-elements/side-menu/_side-menu.scss (hunk 6 above)
Comment (kjugi) [reply components/02-elements/side-menu/_side-menu.scss]: deleted

File: components/02-elements/tab/_tab.scss (hunk 9 above)
Comment (kjugi) [reply components/02-elements/tab/_tab.scss]: deleted

File: components/02-elements/table/_table.scss (hunk 8 above)
Comment (kjugi) [reply components/02-elements/table/_table.scss]: deleted

File: components/02-elements/wrapper-collapsible/_wrapper-collapsible.scss (hunk 7 above)
Comment (kjugi) [reply components/02-elements/wrapper-collapsible/_wrapper-collapsible.scss]: deleted

File: components/03-modules/product/_product.scss (hunk 12 above)
Comment (kjugi) [reply components/03-modules/product/_product.scss]: deleted

File: components/03-modules/products-list-item/_products-list-item.scss (hunk 13 above)
Comment (kjugi) [reply components/03-modules/products-list-item/_products-list-item.scss]: deleted

File: components/03-modules/search/_search.scss (hunk 14 above)
Comment (kjugi) [reply components/03-modules/search/_search.scss]: deleted

File: components/04-views/cart/_discount.scss (hunk 15 above)
Comment (kjugi) [reply components/04-views/cart/_discount.scss]: deleted

File: components/04-views/cart/_main.scss (hunk 16 above)
Comment (kjugi) [reply components/04-views/cart/_main.scss]: deleted

File: components/04-views/catalog/_catalog.scss (hunk 17 above)
Comment (kjugi) [reply components/04-views/catalog/_catalog.scss]: deleted

File: components/02-elements/logo/_logo.scss (hunk 11 above)
Comment (kjugi) [reply components/02-elements/logo/_logo.scss]: deleted

File: components/02-elements/banner/_banner.scss [hunk 37]
```diff
@@ -1,8 +1,31 @@
-$banner-default-position-value: 10%;
-$banner-top-value             : $banner-default-position-value;
-$banner-bottom-value          : $banner-default-position-value;
-$banner-left-value            : $banner-default-position-value;
-$banner-right-value           : $banner-default-position-value;
+$banner-default-position-value             : 10% !default;
+$banner-min-hegight                        : 250px !default;
```
[the remark this answers is not in the document]
Comment (kjugi) [reply components/02-elements/banner/_banner.scss]: fixed

File: components/02-elements/banner/_banner.scss [hunk 38]
```diff
@@ -1,8 +1,31 @@
...
+//content place
+$banner-top-value                          : $banner-default-position-value !default;
+$banner-bottom-value                       : $banner-default-position-value !default;
+$banner-left-value                         : $banner-default-position-value !default;
+$banner-right-value                        : $banner-default-position-value !default;
+
+//content-below-image
+$banner-content-below-background           : $white !default;
+
+//without-image
```
[the remark this answers is not in the document]
Comment (kjugi) [reply components/02-elements/banner/_banner.scss]: ok, space added

File: components/02-elements/list/_list.scss [hunk 39]
```diff
@@ -14,7 +16,10 @@ $list-icon-size          : 16px !default;
 $list-image-width        : 16px !default;
 
 // Divided
-$list-divider            : 1px solid $gray-dark !default;
+$list-divider-color      : $gray-dark !default;
+$list-divider            : 1px solid $list-divider-color !default;
+
+$list-breakpoint-small   : $screen-m !default;
```
[the remark this answers is not in the document]
Comment (kjugi) [reply components/02-elements/list/_list.scss]: deleted

File: components/02-elements/dropdown-list/_dropdown-list.scss (hunk 1 above)
Comment (kjugi) [reply components/02-elements/dropdown-list/_dropdown-list.scss]: changed and checked in other files

File: components/02-elements/form/select/_select.scss (hunk 3 above)
Comment (kjugi) [reply components/02-elements/form/select/_select.scss]: changed and checked in each file

File: components/02-elements/breadcrumbs/_breadcrumbs.scss [hunk 40]
```diff
@@ -1,8 +1,9 @@
-$breadcrumbs-padding        : 10px;
-$breadcrumbs-margin         : 10px;
-$breadcrumbs-color          : $font-color-base; 
-$breadcrumbs-color-second   : $gray;
-$breadcrumbs-clickable-area : 48px;
+$breadcrumbs-padding       : 10px !default;
+$breadcrumbs-margin        : 10px !default;
+$breadcrumbs-color         : $font-color-base !default;
+$breadcrumbs-color-second  : $gray !default;
```
[the remark this answers is not in the document]
Comment (kjugi) [reply components/02-elements/breadcrumbs/_breadcrumbs.scss]: changed `$breadcrumbs-color` to `$breadcrumbs-item-color`
and `$breadcrumbs-color-second ` to `$breadcrumbs-color`

File: components/02-elements/form/input/_input.scss [hunk 41]
```diff
@@ -1,18 +1,20 @@
-$input-border-color     : #d9d9d9 !default;
-$input-padding          : 10px 15px !default;
-$input-border-radius    : $border-radius !default;
-$input-placeholder-color: $gray-light !default;
+$input-margin             : 10px !default;
+$input-padding            : 10px 15px !default;
+$input-field-border-color : #d9d9d9 !default;
```
[the remark this answers is not in the document]
Comment (kjugi) [reply components/02-elements/form/input/_input.scss]: deleted

File: components/02-elements/form/checkbox/_checkbox.scss [hunk 42]
```diff
@@ -1,12 +1,17 @@
...
-$checkbox-icon-size         : 15px !default;
-$checkbox-icon-border-size  : 23px !default;
-$checkbox-icon-border-radius: 2px !default;
+$checkbox-margin             : 10px !default;
+$checkbox-line-height        : $font-line-height !default;
+$checkbox-border-color       : #d9d9d9 !default;
+$checkbox-label-padding      : 33px !default;
+$checkbox-label-before-width : 23px !default;
+$checkbox-label-before-height: 23px !default;
+$checkbox-label-before-border: 1px solid $checkbox-border-color !default;
```
[the remark this answers is not in the document]
Comment (kjugi) [reply components/02-elements/form/checkbox/_checkbox.scss]: ok, `before` deleted

File: components/02-elements/message/_message.scss [hunk 43]
```diff
@@ -1,30 +1,35 @@
...
+$message-background-error  : $red !default;
+$message-background-success: $green !default;
+$message-color             : $white !default;
+$message-color-red         : $red !default;
+$message-color-green       : $green !default;
+$message-padding-base      : 15px !default;
+$message-margin-base       : 15px !default;
+$message-after-height      : 100% !default;
+$message-after-width       : 30px !default;
+$message-success-content   : "\2713" !default;
```
[the remark this answers is not in the document]
Comment (kjugi) [reply components/02-elements/message/_message.scss]: corrected, and add missing letter in variable

File: components/02-elements/minicart-trigger/_minicart-trigger.scss [hunk 44]
```diff
@@ -26,15 +30,15 @@ $minicart-trigger-counter-color     : $white !default;
 
     &__counter {
         position: absolute;
+        display: block;
```
[the remark this answers is not in the document]
Comment (kjugi) [reply components/02-elements/minicart-trigger/_minicart-trigger.scss]: deleted

File: components/02-elements/list/_list.scss [hunk 45]
```diff
@@ -3,6 +3,8 @@ $list-margin             : 10px 0 !default;
 $list-padding            : 0 !default;
 $list-item-padding       : 5px 0 !default;
 $list-content-padding    : 10px !default;
+$list-label-padding      : 0 5px 5px 0 !default;
+$list-value-padding      : 0 5px 5px 0 !default;
```
[the remark this answers is not in the document]
Comment (kjugi) [reply components/02-elements/list/_list.scss]: left it for now

File: components/02-elements/table/_table.scss (hunk 20 above)
Comment (kjugi) [reply components/02-elements/table/_table.scss]: what do you mean?

File: components/02-elements/table/_table.scss (hunk 21 above)
Comment (kjugi) [reply components/02-elements/table/_table.scss]: spaces removed

File: components/03-modules/popup/_popup.scss (hunk 24 above)
Comment (kjugi) [reply components/03-modules/popup/_popup.scss]: rmeoved

File: components/03-modules/popup/_popup.scss (hunk 25 above)
Comment (kjugi) [reply components/03-modules/popup/_popup.scss]: removec

File: components/03-modules/product/_product.scss (hunk 26 above)
Comment (kjugi) [reply components/03-modules/product/_product.scss]: changed

File: components/03-modules/products-list-item/_products-list-item.scss (hunk 27 above)
Comment (kjugi) [reply components/03-modules/products-list-item/_products-list-item.scss]: thanks, fixed

File: components/03-modules/products-list-item/_products-list-item.scss (hunk 28 above)
Comment (kjugi) [reply components/03-modules/products-list-item/_products-list-item.scss]: deleted

File: components/04-views/cart/_main.scss (hunk 29 above)
Comment (kjugi) [reply components/04-views/cart/_main.scss]: deleted

File: components/04-views/cart/_totals.scss (hunk 30 above)
Comment (kjugi) [reply components/04-views/cart/_totals.scss]: deleted

File: components/04-views/catalog/_catalog.scss (hunk 31 above)
Comment (kjugi) [reply components/04-views/catalog/_catalog.scss]: ALL DELETED

File: components/02-elements/banner/_banner.scss [hunk 46]
```diff
@@ -1,8 +1,31 @@
...
+$banner-without-image-background           : $white !default;
+$banner-without-image-height               : 50vh !default;
+$banner-without-image-width                : 100% !default;
```
[the remark this answers is not in the document]
Comment (kjugi) [reply components/02-elements/banner/_banner.scss]: ok, but not in this PR. Too many things to edit. I created a [new issue for this](https://github.com/SnowdogApps/magento2-alpaca-components/issues/102)

File: components/02-elements/banner/_banner.scss [hunk 47]
```diff
@@ -1,8 +1,31 @@
...
-$banner-bottom-value          : $banner-default-position-value;
-$banner-left-value            : $banner-default-position-value;
-$banner-right-value           : $banner-default-position-value;
+$banner-default-position-value             : 10% !default;
+$banner-min-hegight                        : 250px !default;
+$banner-margin                             : 15px !default;
+$banner-padding                            : 15px !default;
+$banner-content-color                      : $gray-dark !default;
+$banner-content-background                 : $white !default;
+$banner-content-background-breakpoint-small: rgba($white, 0.6) !default;
```
[the remark this answers is not in the document]
Comment (kjugi) [reply components/02-elements/banner/_banner.scss]: think it referenced to [this issue](https://github.com/SnowdogApps/magento2-alpaca-components/issues/102)

File: components/02-elements/form/input/_input.scss [hunk 48]
```diff
@@ -36,7 +38,7 @@ $textarea-min-height    : 100px !default;
 
         .input__label {
             margin-bottom: 0;
-            margin-right: 20px;
+            margin-right: ($input-margin * 2);
```
[the remark this answers is not in the document]
Comment (kjugi) [reply components/02-elements/form/input/_input.scss]: nope, changed as it was

File: components/02-elements/form/radio/_radio.scss [hunk 49]
```diff
@@ -29,18 +36,18 @@ $radio-label-active-color: $color-primary;
...
     &__icon {
         position: relative;
         display: inline-block;
-        border: 1px solid $black;
+        border: $radio-icon-border;
         border-radius: $radio-size;
         width: $radio-size;
         height: $radio-size;
-        margin-right: 5px;
+        margin-right: ($radio-margin / 2);
```
[the remark this answers is not in the document]
Comment (kjugi) [reply components/02-elements/form/radio/_radio.scss]: nope, changed as it was

File: components/02-elements/pager/_pager.scss (hunk 18 above)
Comment (kjugi) [reply components/02-elements/pager/_pager.scss]: yup, corrected margin

File: components/02-elements/tab/_tab.scss (hunk 19 above)
Comment (kjugi) [reply components/02-elements/tab/_tab.scss]: ok, this one changed

File: components/04-views/catalog/_catalog.scss (hunk 32 above)
Comment (kjugi) [reply components/04-views/catalog/_catalog.scss]: i changed here some styles b/c it was weird

File: components/04-views/catalog/_catalog.scss (hunk 33 above)
Comment (kjugi) [reply components/04-views/catalog/_catalog.scss]: Generally, I checked all variables with some math and in most cases, I've deleted it. Left only this which i think it's related

File: components/02-elements/table/_table.scss (hunk 20 above)
Comment (kjugi) [reply components/02-elements/table/_table.scss]: ok, done.

File: components/02-elements/cookie-message/_cookie-message.scss (hunk 36 above)
[the remark this answers is not in the document]
Comment (kjugi) [reply components/02-elements/cookie-message/_cookie-message.scss]: ok, changes revert

File: components/02-elements/table/_table.scss (hunk 22 above)
Comment (kjugi) [reply components/02-elements/table/_table.scss]: modifiers moved

File: components/03-modules/form/_form.scss (hunk 23 above)
Comment (kjugi) [reply components/03-modules/form/_form.scss]: modifiers moved
